c2cgeoportal/views/entry.py

```diff
@@ -73,15 +73,17 @@
 
 class DimensionInformation:
 
-    URL_PART_RE = re.compile("[a-zA-Z0-9_\-\+~\.]+$")
+    URL_PART_RE = re.compile("[a-zA-Z0-9_\-\+~\.]*$")
```

Comment:
Why this change?

Reply:
This is a backport of merged commit in 2.3 (support NULL dimensions), because I need it to work with ADL.

Reply:
OK :-)